nflow-engine/src/main/resources/scripts/db/update-5.5.0-x/db2.update.ddl.sql

```diff
@@ -0,0 +1,7 @@
+alter table nflow_workflow add started timestamp(3);
+alter table nflow_archive_workflow add started timestamp(3);
```

Comment:
Directory names for update scripts need to be updated after 5.6.0 release.

Reply:
Done